### DISCUSSION TITLE
Tag not to translate to Google Translate question

### DISCUSSION THREAD

**Discussion by VickiB11** [code4recovery/12-step-meeting-list#1090] [General]
Our website recently added Google Translate so that the Spanish speaking groups can see the TSML meeting finder in Spanish.  However, the meeting location, address, and region are incorrectly translated.  Is there a way to tag the meeting finder not to translate when using Google Translate?

Vicki B
Georgia Al-Anon

![Meetings using Google Translate](https://user-images.githubusercontent.com/74988606/225457591-ec194049-1774-4afc-b0fd-626a5718f7ca.png)

**Comment by joshreisner** [comment]
hi @VickiB11 one option would be to switch your interface to use TSML UI - this translates the meeting finder interface into Spanish, French, Japanese, and Swedish automatically, if your browser is set to prefer one of those languages. you would not need Google Translate for that. it would not change the addresses or other meeting data, just the interface.

**Comment by VickiB11** [comment]
Hi Josh,
Our web developer switched the interface to use TSML-UI.  It looks like that version doesn't provide a filter by District.  Is there any way that can be added with code?

**Reply by joshreisner** [reply]
Hi Vicki, it's not in the same place in the interface as it was, but you can search for them, and you can link to those searches, so you can create your own navigation menu. For example:

* District 01 - links to /meetings?search="District+01"
* District 02 - links to /meetings?search="District+02"

You might add that to the page somewhere, or make a WordPress menu with those links. hope that helps!
